files/build_templates/docker_image_ctl.j2

```diff
@@ -11,6 +11,7 @@ start() {
     else
 
         docker run -d {{docker_image_run_opt}} \
+            --log-opt max-size=2M --log-opt max-file=5 \
```

Comment:
Maximal 5 files, each file maximal 2 MB. So in long term, the disk occupation should be 8 MB ~ 10 MB for each container.